firmware_tools/ghidra/vxhunter_analysis.py

```diff
@@ -1,29 +1,32 @@
+import logging
+import json
+
 from vxhunter_core import *
 from vxhunter_utility.function_analyzer import *
 from vxhunter_utility.symbol import *
 from vxhunter_utility.common import create_initialized_block
 from ghidra.program.model.symbol import RefType, SourceType
-import json
 
 
+
+logger = logging.getLogger(__name__)
```

Comment:
Please don't move the logger, it's make VxAnalyzer support using customized logger.